v2/thirdpartypasswordless/post-login/getting-provider-access-token.mdx

```diff
@@ -33,7 +33,7 @@ SuperTokens.init({
     },
     recipeList: [
         ^{recipeNameCapitalLetters}.init({
-            ^{nodeRecipeInitDefault}
+            ^{nodeRecipeInitDefault} // typecheck-only, removed from output
```

Comment:
You should mention about this special line / comment in the contributing guide too (in the docs/CONTRIBUTING.md)